powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/model/ServiceError.java

```diff
@@ -197,8 +197,59 @@
      */
     public static final String INVALID_ACTIVATION_OTP_MODE = "ERR0032";
 
+    /**
+     * Operation template cannot be found.
+     */
+    public static final String OPERATION_TEMPLATE_NOT_FOUND = "ERR0033";
+
+    /**
+     * Operation cannot be found.
+     */
+    public static final String OPERATION_NOT_FOUND = "ERR0034";
+
+    /**
+     * Operation ID cannot generated.
+     */
+    public static final String UNABLE_TO_GENERATE_OPERATION_ID = "ERR0035";
+
+    /**
+     * Operation is in invalid state for requested change.
+     */
+    public static final String OPERATION_INVALID_STATE = "ERR0036";
+
+    /**
+     * Operation cannot be approved.
+     */
+    public static final String OPERATION_APPROVE_FAILURE = "ERR0037";
+
+    /**
+     * Operation cannot be approved.
+     */
+    public static final String OPERATION_REJECT_FAILURE = "ERR0038";
+
+    /**
+     * Operation template already exists.
+     */
+    public static final String OPERATION_TEMPLATE_ALREADY_EXISTS = "ERR0039";
```

Comment:
Errors 39-42 are missing in documentation.

Reply:
Good catch... we need to redo this error handling one day...

Reply:
Yes, please.